apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -214,6 +303,84 @@ const getResearchOutputRemindersFromQuery = (
   );
 };
 
+const getResearchOutputDraftRemindersFromQuery = (
+  queryResearchOutputsContents: FetchReminderDataQuery['draftResearchOutputs'],
+  findUsersContent: FetchReminderDataQuery['findUsersContent'],
+): ResearchOutputDraftReminder[] => {
+  if (!findUsersContent || !findUsersContent.flatData.teams) {
+    return [];
+  }
+
+  const userTeamIds = getUserTeamIds(findUsersContent.flatData.teams);
+  const userWorkingGroupIds = getUserWorkingGroupIds(
+    findUsersContent.referencingWorkingGroupsContents,
+  );
+
+  if (!queryResearchOutputsContents) {
+    return [];
+  }
+
+  return queryResearchOutputsContents.reduce<ResearchOutputDraftReminder[]>(
+    (researchOutputReminders, researchOutput) => {
+      const userName = getUserName(researchOutput);
+      const { associationName, associationType } =
+        getAssociationNameAndType(researchOutput);
+
+      if (
```

Comment:
Not really happy with this, there must be a better than having this huge if statement